fastly/resource_fastly_ngwaf_workspace.go

```diff
@@ -73,6 +73,11 @@ func resourceFastlyNGWAFWorkspace() *schema.Resource {
 				Description:      "The status code returned when a request is blocked. This configuration is applied at the workspace but can be overwritten in rules. Accepted values are [`301`, `302`, `400..599`]. Default value `406`.",
 				ValidateDiagFunc: validation.ToDiagFunc(validation.Any(validation.IntBetween(400, 599), validation.IntInSlice([]int{301, 302}))),
 			},
+			"default_redirect_url": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "URL to redirect to when blocking with a '301' or '302' HTTP status code",
```

Comment:
```suggestion
				Description: "The redirect URL used if default_blocking_response_code is 301 or 302.",
```

From the public docs.

Reply:
fixed in `83f2419a`